.github/workflows/staging.yaml

```diff
@@ -0,0 +1,34 @@
+name: staging
+
+on:
+  workflow_run:
```

Comment:
I think there are problems with setting up [workflow dependencies](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run).

Even when setting the [github.event.workflow_run.conclusion](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow) flag it does not gather up conclusions from all dependent workflows as suggested in this discussion  https://github.com/orgs/community/discussions/26238

Not to mention that having to set `github.event.workflow_run.conclusion` is kinda ugly and is one of the exact things we wanted to avoid for this refactor.

Reply:
I'm curious, what's the purpose of this particular workflow? It seems to be basically a duplicate of the `release`. Or is this just meant to push non-release builds to Docker Hub?

Basically I'm wondering, how essential is this particular workflow, and how hard do we need to try debugging this particular problem?

Reply:
the purpose is to push a docker container with an `edge` tag. edge tag refers to the latest from master.  this would allow users to run sceptre with the latest code (from master branch).